src/sgame/sg_admin.cpp

```diff
@@ -5047,7 +5057,7 @@ bool G_admin_flag( gentity_t *ent )
 		}
 	}
 
-	if ( !i || flag[ i ] )
+	if ( ( !i || flag[ i ] ) && flag[ 0 ] != '.' )
```

Comment:
This breaks the check for alphanumericness if the string begins with a dot

Reply:
This should be addressed in https://github.com/Unvanquished/Unvanquished/pull/2356/commits/6f103f21639635d8b7460f3af41acdf525dad7e5